### PR TITLE
Some more cleanup and fixes

### PR DESCRIPTION
**Pull request by jflatow**

### CODE REVIEW

File: tests/UniswapConfigTest.js [hunk 1]
```diff
@@ -34,6 +34,24 @@ describe('UniswapConfig', () => {
     await expect(call(contract, 'getTokenConfigByCToken', [address(3)])).rejects.toRevert('revert'); // not a ctoken
   });
 
+  it.only('returns configs exactly as specified', async () => {
```
Comment (coburncoburn) [inline tests/UniswapConfigTest.js:37]: probly dont want to commit the only

File: contracts/AnchoredView/AnchoredView.sol [hunk 2]
```diff
@@ -45,14 +45,13 @@ contract AnchoredView is SymbolConfiguration {
...
     uint immutable upperBoundAnchorRatio;
 
-    /// @notice The lowest ratio of the new median price to the anchor price that will still trigger the median price to be updated
+    /// @dev The lowest ratio of the new median price to the anchor price that will still trigger the median price to be updated
     uint immutable lowerBoundAnchorRatio;
 
-    /// @notice Average blocks per day, for checking anchor staleness
-    /// @dev 1 day / 15
+    /// @dev Average blocks per day, for checking anchor staleness (1 day / 15)
     uint constant blocksInADay = 5760;
```
Comment (coburncoburn) [inline contracts/AnchoredView/AnchoredView.sol:55]: can delete this now I think ( or change message if it is used for something else )

Comment (jflatow) [reply contracts/AnchoredView/AnchoredView.sol:55]: This was in AnchoredView

File: contracts/Uniswap/UniswapAnchoredView.sol [hunk 3]
```diff
@@ -158,34 +164,34 @@ contract UniswapAnchoredView is UniswapConfig {
     }
 
     /**
-     * @dev Fetches the token/eth price accumulator for a config, with same decimals as the token.
+     * @dev Fetches the current token/eth price accumulator from uniswap, with same decimals as the token.
```
Comment (coburncoburn) [inline contracts/Uniswap/UniswapAnchoredView.sol:167]: the decimals for this comment has changed, now it has w/e decimals uniswap normally gives... I guess it looks like it uses decimals from right side of the pair?

Comment (jflatow) [reply contracts/Uniswap/UniswapAnchoredView.sol:167]: Ah yes just saw that in spec but forgot to update the comment here

**Pull request merged**